darwin/torch/transforms.py

```diff
@@ -377,8 +394,20 @@ def _post_process(self, albumentation_output: dict, annotation: dict) -> tuple:
                 output_annotation["masks"] = torch.tensor(np.array(masks))
             else:
                 output_annotation["masks"] = torch.stack(masks)
-            if "area" in annotation:
-                output_annotation["area"] = torch.sum(output_annotation["masks"], dim=[1, 2])
+        elif "masks" in annotation:
+            output_annotation["masks"] = torch.tensor([])
+
+        if "area" in annotation:
+            if "masks" in output_annotation and output_annotation["masks"].numel() > 0:
```

Comment:
stylistic choice: could use `masks` and `bboxes` variables since have them but up to you

Reply:
I think we keep at as it is, masks and bboxes are not the same since we do some conversion.  masks for example is a list, and does not have .numel